docs/operating-scylla/procedures/config-change/switch-snitch.rst

```diff
@@ -1,48 +1,26 @@
 
 How to Switch Snitches
 **********************
-.. REMOVE IN FUTURE VERSIONS - when the limitation in the note is no longer valid
 
 .. note::
 
-    Switching from one type of snitch to another is NOT supported for clusters 
-    where one or more keyspaces have tablets enabled. 
-    
-    NOTE: If you :ref:`create a new keyspace <create-keyspace-statement>`, 
-    it has tablets enabled by default.
+    Changing the network topology by switching to another type of snitch
+    is no longer supported. The new snitch must specify the same DC and rack
+    as the previous one.
```

Comment:
> Remove a note which said that switching snitches is not supported with
> tablets. It was introduced because of the concern that switching a
> snitch might change DC or rack of the node, for which our current tablet
> load balancer is completely unprepated. Now that changing DC/rack is
> forbidden, there doesn't seem to be anything related to snitches which
> could cause trouble for tablets.

@bhalevy while I'm making this claim myself based on what I saw in the discussion under https://github.com/scylladb/scylladb/issues/17513, could you please confirm? I hope I'm not missing anything here - I can bring this note back if needed.

Reply:
Confirmed.  If the snitch change does not change dc/rack, tablets don't care or know about it.